ktor-io/common/test/ByteChannelTest.kt

```diff
@@ -2,152 +2,148 @@
+            channel1.writeByte(42)
         }
     }
 
     @Test
-    fun testCreateFromArray() = testSuspend {
-        val array = byteArrayOf(1, 2, 3, 4, 5)
-        val channel = ByteReadChannel(array)
-        val result = channel.toByteArray()
-        assertTrue(array.contentEquals(result))
+    fun testCreateFromArray() = runTest {
+        val array1 = byteArrayOf(1, 2, 3, 4, 5)
+        val channel1 = ByteReadChannel(array1)
+        val result1 = channel1.toByteArray()
+        assertTrue(array1.contentEquals(result1))
     }
 
     @Test
-    fun testChannelFromString() = testSuspend {
-        val string = "Hello, world!"
-        val channel = ByteReadChannel(string)
-        val result = channel.readRemaining().readText()
-        assertEquals(string, result)
+    fun testChannelFromString() = runTest {
+        val string1 = "Hello, world!"
+        val channel1 = ByteReadChannel(string1)
+        val result1 = channel1.readRemaining().readText()
+        assertEquals(string1, result1)
     }
 
     @Test
-    fun testCancelByteReadChannel() = testSuspend {
-        val channel = ByteReadChannel(byteArrayOf(1, 2, 3, 4, 5))
-        channel.cancel()
+    fun testCancelByteReadChannel() = runTest {
+        val channel1 = ByteReadChannel(byteArrayOf(1, 2, 3, 4, 5))
+        channel1.cancel()
         assertFailsWith<IOException> {
-            channel.readByte()
+            channel1.readByte()
         }
     }
 
     @Test
-    fun testCloseAfterAwait() = testSuspend {
-        val channel = ByteChannel()
-        val job = launch(start = CoroutineStart.UNDISPATCHED) {
-            channel.awaitContent()
+    fun testCloseAfterAwait() = runTest {
+        val channel1 = ByteChannel()
+        val job1 = launch(start = CoroutineStart.UNDISPATCHED) {
+            channel1.awaitContent()
         }
-
-        channel.flushAndClose()
-        job.join()
+        channel1.flushAndClose()
+        job1.join()
     }
 
     @Test
-    fun testChannelMaxSize() = testSuspend(timeoutMillis = 1000) {
-        val channel = ByteChannel()
-        val job = launch(Dispatchers.Unconfined) {
-            channel.writeFully(ByteArray(CHANNEL_MAX_SIZE))
+    fun testChannelMaxSize() = runTest {
+        val channel1 = ByteChannel()
+        val job1 = launch(Dispatchers.Unconfined) {
+            channel1.writeFully(ByteArray(CHANNEL_MAX_SIZE))
         }
-
         delay(100)
-        assertFalse(job.isCompleted)
-
-        channel.readByte()
-        job.join()
+        assertFalse(job1.isCompleted)
+        channel1.readByte()
+        job1.join()
     }
 
     @Test
-    fun testChannelMaxSizeWithException() = testSuspend {
-        val channel = ByteChannel()
-        var writerThrows = false
-        val deferred = async(Dispatchers.Unconfined) {
+    fun testChannelMaxSizeWithException() = runTest {
+        val channel1 = ByteChannel()
+        var writerThrows1 = false
+        val deferred1 = async(Dispatchers.Unconfined) {
```

Comment:
Just curious, what suffix `1` is standing for in these tests?

Reply:
no idea, IntelliJ applied it